codecov.yml

```diff
@@ -13,7 +13,7 @@ ignore:
   - "packages/contracts-bedrock/test/**/*.sol"
   - "packages/contracts-bedrock/scripts/**/*.sol"
   - "packages/contracts-bedrock/contracts/vendor/WETH9.sol"
-  - 'packages/contracts-bedrock/contracts/EAS/**/*.sol'
+  - 'packages/contracts-bedrock/contracts/vendor/eas/**/*.sol'
```

Comment:
This seems out of date. I think the contracts should be in `packages/contracts-bedrock/contracts/src/vendor/eas`.